### PR TITLE
[Manta-PC]Add pallet_scheduler into Manta-pc （Ready to merge)

### LINKED ISSUE #99: Add pallet-scheduler into Manta-pc for parachain runtime upgrade
`pallet-scheduler` allows the parachain to perform some operations after a specified number of blocks have been generated. For example, a runtime upgrade operation is performed after 10 blocks are generated. With `pallet-scheduler`, we can avoid such problems as block weight exhaust.

### PR DESCRIPTION
**Pull request by xialovexia**
Closes #99

### CODE REVIEW

**Comment by ghzlatarev** [comment]
> _No description provided._

Yo, can you add a short description ?
And btw you can link issues from the description by using ``closes #issue_number``

**Comment by xialovexia** [comment]
> > _No description provided._
> 
> Yo, can you add a short description ?
> And btw you can link issues from the description by using `closes #issue_number`

Thanks~ I add it

**Review by ghzlatarev — APPROVED** [review]

File: runtime/manta-pc/src/lib.rs [hunk 1]
```diff
@@ -212,6 +212,27 @@ impl pallet_transaction_payment::Config for Runtime {
...
 }
 
+parameter_types! {
+	// The maximum weight that may be scheduled per block for any
+	// dispatchables of less priority than schedule::HARD_DEADLINE.
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) *
+		RuntimeBlockWeights::get().max_block;
+	// The maximum number of scheduled calls in the queue for a single block.
+	// Not strictly enforced, but used for weight estimation.
+	pub const MaxScheduledPerBlock: u32 = 50;
```
Comment (stechu) [inline runtime/manta-pc/src/lib.rs:222]: why 50?

File: runtime/manta-pc/src/lib.rs [hunk 2]
```diff
@@ -212,6 +212,27 @@ impl pallet_transaction_payment::Config for Runtime {
 	type FeeMultiplierUpdate = SlowAdjustingFeeUpdate<Self>;
 }
 
+parameter_types! {
+	// The maximum weight that may be scheduled per block for any
+	// dispatchables of less priority than schedule::HARD_DEADLINE.
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) *
```
Comment (stechu) [inline runtime/manta-pc/src/lib.rs:218]: why 80?

File: runtime/manta-pc/src/lib.rs (hunk 1 above)
Comment (xialovexia) [reply runtime/manta-pc/src/lib.rs:222]: The parameters here are a reference to the configuration parameters of `pallet_schedule` in `substrate`. Theoretically, it is possible to modify

File: runtime/manta-pc/src/lib.rs (hunk 2 above)
Comment (xialovexia) [reply runtime/manta-pc/src/lib.rs:218]: Similar to the previous

File: runtime/manta-pc/Cargo.toml [hunk 3]
```diff
@@ -50,6 +50,7 @@ pallet-timestamp = { version = '3.0.0', git = 'https://github.com/paritytech/sub
 pallet-transaction-payment = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "master" }
 pallet-transaction-payment-rpc-runtime-api = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "master" }
 pallet-utility = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "master" }
+pallet-scheduler = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
```
Comment (stechu) [inline runtime/manta-pc/Cargo.toml]: Can you specify a version here?

**Review by stechu — CHANGES_REQUESTED** [review]

**Comment by stechu** [comment]
@xialovexia 
Please remove the `.DS_Store` file (this pollutes the git):
https://github.com/2DegreesInvesting/resources/issues/115

File: runtime/manta-pc/Cargo.toml (hunk 3 above)
Comment (xialovexia) [reply runtime/manta-pc/Cargo.toml]: Update

**Pull request merged**